versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = "~> 0.12.6"
+
+  required_providers {
+    aws    = "~> 2.41.0"
+    random = "~> 2.2.1"
```

Comment:
1. Remove `random` and `null` from this list.
2. Replace `aws    = "~> 2.41.0"` with `aws    = "~> 2.41"`

PS: Make it exactly as in VPC module.